From 87091daca5ebbebfa8ed9f8ac317b45ff4b90f51 Mon Sep 17 00:00:00 2001
From: Wenchao Hao <haowenchao@huawei.com>
Date: Tue, 8 Feb 2022 18:39:47 +0800
Subject: [PATCH] Remove iscsi-init.service from iscsi and iscsid service file

iscsi-init.service is aimed to generate initiatorname.iscsi,
this name would be generated by spec file in openeulr, so we
excluded the iscsi-init.service.

This patch remove requires of iscsi-init in iscsi.service

Signed-off-by: Wenchao Hao <haowenchao@huawei.com>
---
 etc/systemd/iscsi.service  | 2 +-
 etc/systemd/iscsid.service | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/systemd/iscsi.service b/etc/systemd/iscsi.service
index 5e394b9..020a207 100644
--- a/etc/systemd/iscsi.service
+++ b/etc/systemd/iscsi.service
@@ -3,7 +3,7 @@ Description=Login and scanning of iSCSI devices
 Documentation=man:iscsiadm(8) man:iscsid(8)
 Before=remote-fs.target
 After=network-online.target iscsid.service
-Requires=iscsid.socket iscsi-init.service
+Requires=iscsid.socket
 Wants=network-online.target
 
 [Service]
diff --git a/etc/systemd/iscsid.service b/etc/systemd/iscsid.service
index 49347f5..9d689e5 100644
--- a/etc/systemd/iscsid.service
+++ b/etc/systemd/iscsid.service
@@ -2,7 +2,7 @@
 Description=Open-iSCSI
 Documentation=man:iscsid(8) man:iscsiadm(8)
 DefaultDependencies=no
-After=network-online.target iscsi-init.service
+After=network-online.target
 Before=remote-fs-pre.target
 Wants=remote-fs-pre.target
 
-- 
2.34.1

